.vitepress/config.mts

```diff
@@ -11,13 +11,21 @@ const files = globSync('./src/**/README.md').map(item => {
     text: name,
     link: item
   }
-}).filter(Boolean)
+}).filter(notEmpty)
+
+function notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
+  return value !== null && value !== undefined
+}
 
 // https://vitepress.dev/reference/site-config
 export default defineConfig({
   title: 'kuma-gui',
   description: '',
   cleanUrls: true,
+  ignoreDeadLinks: [
+    // ignore all localhost links
+    /^https?:\/\/localhost/,
+  ],
```

Comment:
Do you know what links are causing this not to build?

I don't think we would ever want to build this, and I was going to say we may as well just remove that script, but! the build script would be good to use to check that there aren't any broken links (unless you can get it to do the same check on `docs:dev`?).

Reply:
It’s the references in DEVELOPER.md and they’re perfectly fine there _but_ will of course fail any link check test.